swagger/api.spec.yaml

```
@@ -440,25 +440,25 @@ paths:
       operationId: api.group.patch_group_by_id
       tags:
         - groups
-      summary: Merge group information
+      summary: Update group information.
       description: >-
-        Merge group information.
+        Add information to a group removing existing associations to the group.
         <br /><br />
         Required permissions: inventory:groups:write
       security:
         - ApiKeyAuth: []
       parameters:
         - $ref: '#/components/parameters/groupId'
       requestBody:
-        description: A dictionary with new information to merge with the original group.
+        description: A dictionary with new information to update the original group.
```

Just a little grammar suggestion
```suggestion
        description: A dictionary with new information with which to update the original group.
```